specification/authorization/resource-manager/Microsoft.Authorization/preview/2024-02-01-preview/RoleManagementPolicyAssignment.json

```diff
@@ -0,0 +1,393 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the role management policy assignment to delete.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "name": "roleManagementPolicyAssignmentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of format {guid_guid} the role management policy assignment to delete."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Successfully deleted the policy assignment."
+          },
+          "204": {
+            "description": "NoContent - policy assignment does not exists."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeleteRoleManagementPolicyAssignment": {
+            "$ref": "./examples/DeleteRoleManagementPolicyAssignment.json"
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Authorization/roleManagementPolicyAssignments": {
+      "get": {
+        "tags": [
+          "roleManagementPolicyAssignments"
+        ],
+        "operationId": "RoleManagementPolicyAssignments_ListForScope",
+        "description": "Gets role management assignment policies for a resource scope.",
+        "parameters": [
+          {
+            "name": "scope",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The scope of the role management policy.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of role management policies.",
+            "schema": {
+              "$ref": "#/definitions/RoleManagementPolicyAssignmentListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetRoleManagementPolicyAssignmentByScope": {
+            "$ref": "./examples/GetRoleManagementPolicyAssignmentByScope.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RoleManagementPolicyAssignment": {
+      "properties": {
+        "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

Please use $ref to Resource (or ProxyResource) definition from v5 common types for resource envelope properties (id, name, type).